voxblox/src/utils/camera_model.cc

```diff
@@ -55,31 +53,31 @@ void CameraModel::setIntrinsicsFromFoV(double horizontal_fov,
   double tan_half_horizontal_fov = std::tan(horizontal_fov / 2.0);
   double tan_half_vertical_fov = std::tan(vertical_fov / 2.0);
   untransformed_corners_.emplace_back(
-      Eigen::Vector3d(min_distance, min_distance * tan_half_horizontal_fov,
-                      min_distance * tan_half_vertical_fov));
+      Point(min_distance, min_distance * tan_half_horizontal_fov,
+            min_distance * tan_half_vertical_fov));
   untransformed_corners_.emplace_back(
-      Eigen::Vector3d(min_distance, min_distance * tan_half_horizontal_fov,
-                      -min_distance * tan_half_vertical_fov));
+      Point(min_distance, min_distance * tan_half_horizontal_fov,
+            -min_distance * tan_half_vertical_fov));
   untransformed_corners_.emplace_back(
-      Eigen::Vector3d(min_distance, -min_distance * tan_half_horizontal_fov,
-                      -min_distance * tan_half_vertical_fov));
+      Point(min_distance, -min_distance * tan_half_horizontal_fov,
+            -min_distance * tan_half_vertical_fov));
   untransformed_corners_.emplace_back(
-      Eigen::Vector3d(min_distance, -min_distance * tan_half_horizontal_fov,
-                      min_distance * tan_half_vertical_fov));
+      Point(min_distance, -min_distance * tan_half_horizontal_fov,
```

Comment:
Ugh thanks for cleaning all these up.

Reply:
Was actually pretty quick just did a find replace for Eigen::Vector3d and Point